packages/web/src/common/store/cache/users/sagas.ts

```diff
@@ -122,16 +127,32 @@ export function* fetchUserByHandle(
  * @param {number} userId target user id
  */
 export function* fetchUserCollections(userId: number) {
-  const audiusBackendInstance = yield* getContext('audiusBackendInstance')
-  // Get playlists.
-  const playlists = yield* call(
-    audiusBackendInstance.getPlaylists,
-    userId,
-    null
-  )
+  const sdk = yield* getSDK()
+  function* getPlaylists() {
+    const { data } = yield* call(
+      [sdk.full.users, sdk.full.users.getPlaylistsByUser],
+      {
+        id: Id.parse(userId)
```

Comment:
I believe this is going to lose the current user context if you don't pass in `userId` for the SDK version.
The ABE versions are sneaky about this, because the current user id is buried all the way down in the function where the request is created, so you aren't required to pass it at this level.

https://github.com/AudiusProject/audius-protocol/blob/018ad17a42d7840ac397faa63106f2c3d2b59f43/packages/libs/src/services/discoveryProvider/DiscoveryProvider.ts#L1651 

And since `withUsers` defaults to true, I _think_ this would normally have returned all the playlists with follower/followee counts etc.